### PR TITLE
Origin/feat/change password

### PR DESCRIPTION
**Pull request by minhdev313**
Nhờ Minh review nhé

### CODE REVIEW

File: internal/controller/auth.controller.go [hunk 1]
```diff
@@ -170,3 +170,33 @@ func (ac *AuthController) ResetPassword(ctx *gin.Context) {
...
+		return
+	}
+
+	statusCode, err := ac.authService.ChangePassword(ctx, &input)
+	if err != nil {
+		response.ErrorResponse(ctx, statusCode, err.Error())
+		return
+	}
+
+	response.SuccessResponse(ctx, http.StatusOK, "Password changed successfully.")
```
Comment (minhbear) [inline internal/controller/auth.controller.go]: Should used i18n instead fix like this, check another code to see example

File: internal/dto/auth_dto/auth.dto.go [hunk 2]
```diff
@@ -25,3 +25,8 @@ type LoginSwaggerOutput struct {
 	Success bool        `json:"message"`
 	Data    LoginOutput `json:"data"`
 }
+
+type ChangePasswordInput struct {
+	OldPassword string `json:"oldPassword" binding:"required"`
```
Comment (minhbear) [inline internal/dto/auth_dto/auth.dto.go]: json:"old_password"
json:"new_password"

File: internal/service/auth/auth.service.go [hunk 3]
```diff
@@ -288,3 +290,46 @@ func (as *authService) clearTokenSessions(ctx *gin.Context, email string) error
 
 	return nil
 }
+
+func (s *authService) ChangePassword(ctx *gin.Context, input *auth_dto.ChangePasswordInput) (int, error) {
+	currentUser := context_util.GetUserContext(ctx)
+	fmt.Println(currentUser.ID)
```
Comment (minhbear) [inline internal/service/auth/auth.service.go]: remove fmt.Println

File: internal/service/auth/auth.service.go [hunk 4]
```diff
@@ -288,3 +290,46 @@ func (as *authService) clearTokenSessions(ctx *gin.Context, email string) error
...
+	}
+
+	if err := global.Db.Model(&user).Where("id = ?", currentUser.ID).Update("password", hashedNewPassword).Error; err != nil {
+		message := global.Localizer.MustLocalize(&i18n.LocalizeConfig{
+			MessageID: constant.MessageI18nId.InternalServerError,
+		})
+		return http.StatusInternalServerError, errors.New(message)
+	}
+
+	global.Logger.Info("User password changed successfully", zap.Int64("userID", user.ID))
```
Comment (minhbear) [inline internal/service/auth/auth.service.go]: Remove unnecessary logger in here

File: pkg/utils/jwt/jwt.util.go [hunk 5]
```diff
@@ -25,6 +25,10 @@ type InviteMentorJwtInput struct {
 	InviteID        int64
 }
 
+type ChangePasswordJwtInput struct {
```
Comment (minhbear) [inline pkg/utils/jwt/jwt.util.go]: Why need this, where to used this ?

File: internal/controller/auth.controller.go [hunk 6]
```diff
@@ -170,3 +170,33 @@ func (ac *AuthController) ResetPassword(ctx *gin.Context) {
 
 	response.SuccessResponse(ctx, statusCode, "")
 }
+
+// @Summary ChangePassword
```
Comment (minhbear) [inline internal/controller/auth.controller.go:174]: Had run make swagger to update the swagger docs ?, I did not see any change in docs folder, should run make swagger to update swagger docs

File: internal/controller/auth.controller.go (hunk 1 above)
Comment (minhdev313) [reply internal/controller/auth.controller.go]: I have fixed

File: internal/dto/auth_dto/auth.dto.go (hunk 2 above)
Comment (minhdev313) [reply internal/dto/auth_dto/auth.dto.go]: I have fixed

File: pkg/utils/jwt/jwt.util.go (hunk 5 above)
Comment (minhdev313) [reply pkg/utils/jwt/jwt.util.go]: I have removed

File: internal/service/auth/auth.service.go (hunk 3 above)
Comment (minhdev313) [reply internal/service/auth/auth.service.go]: I have removed

File: internal/service/auth/auth.service.go (hunk 4 above)
Comment (minhdev313) [reply internal/service/auth/auth.service.go]: i have removed

**Pull request merged**